ru/lessons/libraries/benchee.md

```diff
@@ -0,0 +1,326 @@
+* **parallel** &mdash; количество процессов, которое следует использовать при тестировании. Так, если мы передадим `parallel: 4`, то создадутся 4 процесса, каждый из которых будет выполнять одну и ту же функцию на протяжении `time` секунд. Когда они закончат, 4 новых процесса будут запущены уже для следующей функции. Это позволит собрать больше данных за то же время, но при этом создаст некоторую нагрузку на систему, что может повлиять на результаты. Этот параметр можно использовать для симуляции системы под нагрузкой, что иногда полезно. Однако надо помнить, что параллельное выполнение может непредсказуемо повлиять на результаты, так что использовать его нужно с осторожностью. Значение по умолчанию: 1 (т.е. без параллельного выполнения).
+* **formatters** &mdash; форматтеры &mdash; список форматирующих функций, который вы бы хотели использовать для отображения результата выполнения `Benchee.run/2`. Функции должны принимать один аргумент (набор данных) и затем с его помощью отображать результат. По умолчанию используется встроенный консольный форматтер `Benchee.Formatters.Console.output/1`. Мы подробнее рассмотрим форматтеры чуть позже.
+* **print** &mdash; ассоциативный массив или ключевой список со следующими опциями-атомами в качестве ключей и `true` или `false` в качестве значений. С помощью этого параметра мы можем указывать, какие метрики должны быть выведены в ходе стандартного выполнения теста. Все опции включены по умолчанию (имеют значение `true`). Вот они:
+  * **benchmarking** &mdash; выводится, когда `Benchee` запускает новый тест.
+  * **configuration** &mdash; краткий список всех настроек, включая прогнозируемое время выполнения. Выводится перед началом теста.
+  * **first_warning** &mdash; предупреждения выводятся в случае, если функция выполняется слишком быстро, что потенциально может привести к неточным результатам.
+* **console** &mdash; ассоциативный массив или ключевой список со следующими опциями-атомами в качестве ключей и произвольными значениями. Доступные опции:
+  * **comparison** &mdash; показывать ли сравнение результатов разных тестов (*х* в столько-то раз медленее, чем *y*). По умолчанию `true`, но можно отключить при помощи `false`.
+  * **unit_scaling** &mdash; масштабирование &mdash; стратегия выбора единиц измерения для времени и количества. Когда масштабирование включено, `Benchee` подыщет наиболее подходящее отображение для чисел. Например, `1_200_000` масштабируется в 1,2 М, а `800_000` в 800 К. Стратегия масштабирования определяет то, как `Benchee` будет выбирать отображение числовых значений в случаях, когда "наилучший" вид разный для разных чисел. Всего есть четыре стратегии, у каждой &mdash; имя-атом. По умолчанию используется `:best`:
+    * **best** &mdash; будет выбрано наиболее часто встречающееся сокращение. В случае совпадения, будет выбрано наибольшее.
+    * **largest** &mdash; среди наилучших сокращений будет выбрано наибольшее
```

Comment:
All the list items except for those on lines 132-133 are ended with a period. I think it would be nice to be consistent and add periods to those ones too.